### PR TITLE
Add SSR

### PR DESCRIPTION
**Pull request by cedricdelpoux**

### CODE REVIEW

File: example/server/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+process.env.BABEL_ENV = "development"
```
[the remark this answers is not in the document]
Comment (myagoo) [reply example/server/bootstrap.js]: Or not use it at all ?

File: example/src/App.js [hunk 2]
```diff
@@ -137,17 +131,27 @@ export default function App() {
...
+            <Text css={{flex: ["1", "none"]}}>{id}</Text>
             <button onClick={() => setItems(items.filter(ts => ts !== id))}>
               x
             </button>
           </View>
-        );
+        )
       })}
       <button onClick={() => setItems([...items, Date.now()])}>Add view</button>
+      <Text
```
[the remark this answers is not in the document]
Comment (cedricdelpoux) [reply example/src/App.js]: Yes, to seee if SSR works if javascript is disabled

File: src/stylesheet.tsx [hunk 3]
```diff
@@ -34,4 +32,12 @@ class ServerStyleSheet {
   }
 }
 
+const isBrowser = typeof document !== "undefined"
+
+const StyleSheetContext: Context<
+  ServerStyleSheet | ClientStyleSheet
+> = React.createContext(
+  isBrowser ? new ClientStyleSheet() : new ServerStyleSheet()
```
Comment (cedricdelpoux) [inline src/stylesheet.tsx]: Nice move 👍🏻

File: example/server/bootstrap.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cedricdelpoux) [reply example/server/bootstrap.js]: I need to define them
`Using `babel-preset-react-app` requires that you specify `NODE_ENV` or `BABEL_ENV` environment variables. Valid values are "development", "test", and "production"`

**Pull request merged**